checkov/common/checks_infra/solvers/attribute_solvers/base_attribute_solver.py

```diff
@@ -45,6 +46,24 @@ def run(self, graph_connector: DiGraph) -> Tuple[List[Dict[str, Any]], List[Dict
         passed_vertices: List[Dict[str, Any]] = []
         failed_vertices: List[Dict[str, Any]] = []
         unknown_vertices: List[Dict[str, Any]] = []
+        if isinstance(graph_connector, Graph):
+            if self.resource_types:
+                select_kwargs = {"resource_type_in": self.resource_types}
+            else:
+                select_kwargs = {"block_type__in": SUPPORTED_BLOCK_TYPES}
+
+            for data in graph_connector.vs.select(**select_kwargs)["attr"]:
+                result = self.get_operation(vertex=data)
+                # A None indicate for UNKNOWN result - the vertex shouldn't be added to the passed or the failed vertices
+                if result is None:
+                    unknown_vertices.append(data)
+                elif result:
+                    passed_vertices.append(data)
+                else:
+                    failed_vertices.append(data)
```

Comment:
multi threading wouldn't help, because it is CPU bound, only multiprocessing could give some boost. If we need it, then we should do it in a separate PR and it could become tricky, because we also leverage multiprocessing on the runner level.